notebooks/feature_engineering/raw/ex1.ipynb

```diff
@@ -227,7 +221,9 @@
     "# Your code here: create train, validation, and test sets as dataframes\n",
     "train = ____\n",
     "valid = ____\n",
-    "test = ____"
+    "test = ____\n",
+    "# Run this cell to check your work\n",
```

Comment:
This split felt a little fiddly... time consuming to get the details to match expectations, but not conceptually hard or interesting. I think we should just give the users this code rather than making it an exercise.

Reply:
Removing this question is fine.